### PR TITLE
Add error messages to the empty try-except blocks

### PR DESCRIPTION
**Pull request by banasiakadam60**
#667 
Signed-off-by: Adam Banasiak <banasiakadam60@gmail.com>

### CODE REVIEW

File: golem/http/stream.py [hunk 1]
```diff
@@ -305,8 +305,10 @@ def __disconnect(self):
             # read remaining data
             try:
                 self.__read(drain=True)
-            except:
-                pass
+            except StopIteration as ste:
```
Comment (mfranciszkiewicz) [inline golem/http/stream.py]: `StopIteration` will always be raised while working on valid streams; I think we don't need a debug log here.

File: gui/application.py [hunk 2]
```diff
@@ -16,8 +16,8 @@ def __init__(self, app_logic, mainWindowClass):
         try:
             # Linux check might suffice if X11 was the only option available
             QtCore.QCoreApplication.setAttribute(QtCore.Qt.AA_X11InitThreads)
-        except Exception:
-            pass
+        except Exception as ex:
+            logger.warning("Error occurred when setting up Qt: {}".format(ex.message))
```
Comment (mfranciszkiewicz) [inline gui/application.py]: This block will always fail on Windows.

**Comment by badb** [comment]
Can you change printing / logging to print exception rather than exception message? Not all exceptions have properly set "message" variable.

**Comment by banasiakadam60** [comment]
Done ;)

**Pull request merged**